cmd/repl/main.go

```diff
@@ -244,7 +251,23 @@ func put(parent *kingpin.Application, gdb gdb, in io.Reader) {
 		if _, err := v.ReadFrom(in); err != nil {
 			return err
 		}
-		return db.Put(*id, v.Bytes())
+
+		tx := db.NewTransaction()
```

Comment:
There's no need to implement this because the `put` feature of repl must go away (because we'd need a server side endpoint to call to make it meaningful).

Reply:
Lets remove it when we fix rebase